social-messenger-ts/src/assets/index.ts

```diff
@@ -66,27 +66,38 @@ const randomImages = [
   avatar24,
 ];
 
-export const getRandomImage = () => {
-  const index = Math.floor(Math.random() * 24);
-  return randomImages[index];
+export const getImage = (userId: string) => {
+  const hash = hashCode(userId);
+  const index = Math.abs(hash) % staticImages.length;
```

Comment:
```suggestion
  const index = Math.abs(hash) % staticImages.length + 1;
```

Without the `+ 1` you'll never be able to access the last image of the array as the returned index will always be `i >= 0 && i < array.length` - to make sure I tested it with this little code snippet:
![image](https://user-images.githubusercontent.com/43254280/173550414-593a9998-f2ec-43c7-98c8-a8e5a2b20d37.png)

For `100000` elements and `100` passes it always returned `false`.